src/ResourceManager/Network/Commands.Network/VirtualNetworkGateway/NewAzureVirtualNetworkGatewayCommand.cs

```diff
@@ -149,7 +149,8 @@ public class NewAzureVirtualNetworkGatewayCommand : VirtualNetworkGatewayBaseCmd
             HelpMessage = "The list of P2S VPN client tunneling protocols")]
         [ValidateSet(
             MNM.VpnClientProtocol.SSTP,
-            MNM.VpnClientProtocol.IkeV2)]
+            MNM.VpnClientProtocol.IkeV2,
+            MNM.VpnClientProtocol.OpenVPN)]
```

Comment:
It seems the Models do not have the value `OpenVPN`, is this something that is not currently released?

Reply:
@praries880 This is why we are merging to this particular branch instead of preview branch. @MikhailTryakhov will eventually import the updated nuget network SDK into this, which will contain the updated model.